app/page/template/panel/guest-options.htm

```diff
@@ -1,9 +1,36 @@
-<div  id="guest-options" class="guest-options" data-bind="with: $root.guestOptions">
+<div id="guest-options" class="panel__page panel__page--visible guest-options" data-bind="with: $root.guestOptions">
```

Comment:
Technically fine but only the default should have the visible class. Will be handled by the view model.